.github/pull_request_template.md

```diff
@@ -15,13 +15,17 @@ Please describe the tests you've added or any tests that already cover this chan
 
 ### Checklist:
 
-- [ ] My code follows the style guidelines of this project
-- [ ] I have performed a self-review of my own code
-- [ ] I have commented my code, particularly in hard-to-understand areas
-- [ ] I have updated the release notes
-- [ ] I have made corresponding changes to the documentation
-- [ ] I have added debugging support to my changes
-- [ ] I have added tests that prove my fix is effective or that my feature works
-- [ ] New and existing unit tests pass locally with my changes
-- [ ] I have added any new baselines to the repo
-- [ ] I have assigned reviewers (see [VisIt's PR procedures](https://visit-sphinx-github-user-manual.readthedocs.io/en/develop/dev_manual/pr_create.html#reviewers) for more information).
+- [ ] My code follows the [style guidelines][1] of this project.~~
+- [ ] I have performed a self-review of my own code.~~
+- [ ] I have commented my code, particularly in hard-to-understand areas.~~
```

Comment:
This can be changed to, "I have commented my code where applicable", or something like that, to clarify that comments don't always have to be provided, but should be provided in areas where the code may be a bit confusing.